Mathlib/Topology/MetricSpace/Lipschitz.lean

```diff
@@ -581,6 +590,135 @@ end Metric
 
 end LipschitzOnWith
 
+namespace LocallyLipschitz
+variable [PseudoEMetricSpace α] [PseudoEMetricSpace β] [PseudoEMetricSpace γ] {f : α → β}
+
+/-- A Lipschitz function is locally Lipschitz. -/
+protected lemma _root_.LipschitzWith.locally {K : ℝ≥0} (hf : LipschitzWith K f) :
+    LocallyLipschitz f :=
+  fun _ ↦ ⟨K, ⟨univ, ⟨Filter.univ_mem, Iff.mpr (lipschitzOn_univ) hf⟩⟩⟩
```

Comment:
Angle brackets nest implicitly, so you can reduce this to:
```suggestion
  fun _ ↦ ⟨K, univ, Filter.univ_mem, lipschitzOn_univ.mpr hf⟩
```

Reply:
:bulb: :exploding_head: Wow, that's even nicer!